server/hotrod/src/test/resources/create_certs.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+echo "==== Removing old files ===="
+find . -name "*.jks" -exec rm -rf {} \;
+
+echo "==== Generating private keys ===="
+keytool -genkey -noprompt -trustcacerts -keyalg RSA -alias "default" -dname "CN=HotRod_1, OU=Infinispan, O=JBoss, L=Red Hat, ST=World, C=WW" -keypass "secret" -storepass "secret" -keypass "secret2" -keystore "password_server_keystore.jks"
```

Comment:
While we're here, we might as well introduce a more articulate test keystore using the following hierarchy
Root CA > CA > Certificates*